core/controllers/classroom.py

```diff
@@ -151,11 +151,32 @@ class ClassroomIdToNameHandler(
     @acl_decorators.open_access
     def get(self) -> None:
         """Retrieves a mapping of classroom IDs to classroom names."""
-        classroom_id_to_classroom_name = (
-            classroom_config_services.get_classroom_id_to_classroom_name_dict())
+        classroom_id_index_mappings: List[Dict[str, str|int]] = []
+        classrooms = classroom_config_services.get_all_classrooms()
+
+        for classroom in classrooms:
+            classroom_id_index_mapping_dict: Dict[str, str | int]
```

Comment:
Nit on line 158: `str | int` -> `str|int` (for consistency with line 154)

Reply:
Done.